keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -459,87 +464,136 @@ Windows privileged containers will be implemented with [Job Objects](https://doc
+- Directory mounts (i.e. directory C:\test mapped to C:\test in the privileged container) are still being investigated. Note that file system mapping in the current implementation would expose both the host filesystem and the container filesystem to the processes in the privileged container. We are still investigation how external mounts such as secrets or storage accounts would be added. The default visibility of all the host filesystem may present backwards compatibility challenges in the future if host path mounting is enabled in Windows. Currently the only method to restrict would be via the user in a way similar to [UID or GID](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) in the PSP. Currently however, this host path mounting is unlikely to be enabled in the future as a restriction of Job Objects.
+
 #### Container Images
-- A slim base image may be required to satisfy hierarchy requirements from HCS. It was found that the graphdriver calls expect certain files to be in a windows image when un-tarring. These files were found to be simply several registry hive files in /windows/system32/config (which can be empty).
-- We are currently targeting to release a simple image alongside this feature including the above mentioned files, which can be used instead of traditional Windows base images of server core or otherwise. Server core and others that container the necessary files would also work as privileged container base images. 
+
+- Privileged containers can be built on top of existing Windows base images (nanoserver, servercore, etc).
+- A slim base image may be introduced to act as a replacement for [scratch]. On windows graphdriver calls expect some files (mainly registry hives) and these would be included in the slim base image.
+- Privileged containers will not inherit the same [compatibility requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility) as process isolated containers from an OS perspective. Container runtimes like containerd may need to skip OS version checks when pulling/starting these containers. This will continue to be investigated as the feature matures.
```

Comment:
How  would containerd know whether to skip checks when pulling an image?

Reply:
If we are going to add `privileged` into `PodSandboxConfig`, containerd would know it at image pulling time.

Reply:
Correct, once we have the fields in the CRI calls we would add a check into containerd to skip OS version validation.

Reply:
Added a note in the KEP.